src/fieldFactory.js

```diff
@@ -26,13 +36,14 @@ export default (field, options) => {
     );
   }
 
-  if (null !== field.reference) {
+  if (null != field.reference) {
```

Comment:
Changed to check for undefined

Reply:
I've dropped this commit, the `field.reference` should be null of have a value, there must be something wrong if it is undefined. Can you give steps to reproduce ?

Reply:
Yes you are right, I made this change while I was missing your last commits.